x/evm/keeper/querier.go

```diff
@@ -81,6 +81,20 @@ func GetCommandResponse(ctx sdk.Context, chainName string, n types.Nexus, cmd ty
 	params := make(map[string]string)
 
 	switch cmd.Command {
+	case types.AxelarGatewayCommandApproveContractCallWithMint:
+		sourceChain, sourceAddress, contractAddress, payloadHash, symbol, amount, err := types.DecodeApproveContractCallWithMintParams(cmd.Params)
```

Comment:
suggestions: add a `DecodeParams()` function to the command itself. Then we can use an interface and make this query generic so we don't have to touch it every time a new command is added